Evolutionary_loop/Loop_Parts/Part_D/part_d_job1.py

```diff
@@ -0,0 +1,114 @@
+#*******************************************************************************
+# File: part_d_job1.py
+#    Based on Part_D1_Array.sh.
+#
+#  Programmer: Jason Yao (yao.966@osu.edu)
+#
+#  Revision history:
+#     07/28/23 Jason Yao, Original version
+#
+#  Notes:
+#       * vertical ruler at column 80
+#  TODO:
+#       * Incorporate process 0 into part_c.py
+#       * p1 needs to be combined with the sbatch command (ie. in the same
+#         subprocess)
+#       * remove .root files
+#
+#*******************************************************************************
+'''
+This is Part D1 of the Loop.
+This script moves each .dat file (from Part C) into a folder AraSim can access,
+while changing it to a .txt, which is what AraSim reads. For each individual,
+the script then runs AraSim and moves the output into Antenna_Performance_Metric.
+Finally, this script makes a directory for all errors and output files from
+AraSim to be dumped.
+
+Usage: python4 part_d_job1.py <indiv> <gen> <npop> <working_dir> ... <nsections>
+
+For more information on the arguments: 
+  >> python3 part_d_job1.py -h
+'''
+
+
+import argparse
+import subprocess as sp
+
+from pathlib import Path
+
+
+def main(gen, npop, working_dir, arasim_exec,
+         exp, nnt, run_name, seeds, debug_mode):
+    
+## AraSim SETUP
+    specific_seed = 32000
+
+    metric_path = working_dir / 'Antenna_Performance_Metric'
+    for i in range(1,npop+1):
+        p0 = sp.run(f'mv {metric_path}/evol_antenna_model_{i}.dat '
+                    f'{arasim_exec}/a_{i}.txt', shell=True, capture_output=True)
```

Comment:
Here you should use `pathlib.Path.rename`

Reply:
Actually `p0` should be deleted since the process can be combined with `part_c.py`